package/MDAnalysis/analysis/rms.py

```diff
@@ -484,6 +486,63 @@ def save(self, filename=None):
         return filename
 
 
+def rmsf(atomgroup, start=0, stop=-1, step=1, progout=None):
```

Comment:
As you probably saw `rmsd()` does not work on a whole trajectory but only on simple coordinates (btw, raise an issue for adding the option not to do fitting). Generally, anything that takes trajectories as inputs *should* be a class with a common interface of init and run (and perhaps analyze, plot, save...), at least that's the current informal consensus. In the longer term we should be able to integrate this back into the [`correl`/`timeseries`](http://pythonhosted.org/MDAnalysis/documentation_pages/core/Timeseries.html) approach where you can stack up all the analysis you like and then run through the trajectory only once. (We'll then only simply need to add additional per-step methods to the analysis classes to make this work.)

Bottomline: I'd really think this should go in as a class, even it's only the basic functionality, but it will be easier to add bells and whistles later. If you don't have the time to do it we'll retarget to a later milestone.